core/storage/question/gae_models.py

```diff
@@ -66,6 +66,9 @@ class QuestionModel(base_models.VersionedModel):
     # The skill ids linked to this question.
     linked_skill_ids = ndb.StringProperty(
         indexed=True, repeated=True)
+    # The optional misconception ids marked as not relevant to the question.
+    inapplicable_misconception_ids = ndb.StringProperty(
+        indexed=True, repeated=True)
```

Comment:
Do we need to have a one-off job to up[date the existing models?

Reply:
As mentioned in https://github.com/oppia/oppia/pull/10481#issuecomment-683319706, we do not require a one-off job to update existing models. It will automatically get set to `[]`.

Reply:
I see thanks for the link. One question: these values are marked `indexed` so will it be possible to do a query using this field without any update? [Like you want to fetch all the model with empty `inapplicable_misconception_ids`]

Reply:
As per our offline discussion, there isn't any way to query for an empty list on age, marking this resolved!